actions/crib-deploy-environment/action.yml

```diff
@@ -128,8 +153,8 @@ runs:
         CRIB_SKIP_DOCKER_ECR_LOGIN: true
         CRIB_SKIP_HELM_ECR_LOGIN: true
         DEVSPACE_IMAGE:
-          "${{
-          inputs.ecr-private-registry-stage}}.dkr.ecr.us-west-2.amazonaws.com/chainlink-devspace"
+          "${{ inputs.ecr-private-registry-stage
+          }}.dkr.ecr.us-west-2.amazonaws.com/chainlink-devspace"
```

Comment:
Will this line break create an issue?

Reply:
I have tested but, but will fix it.

Reply:
It was already multiline :)If I add a space to the parentheses, it has to be multiline because the Prettier hook complains.